bigbluebutton-html5/imports/api/meetings/server/methods/endMeeting.js

```diff
@@ -2,13 +2,34 @@ import { Meteor } from 'meteor/meteor';
 import RedisPubSub from '/imports/startup/server/redis';
 import Logger from '/imports/startup/server/logger';
 import { extractCredentials } from '/imports/api/common/server/helpers';
+import Meetings from '/imports/api/meetings';
+import Users from '/imports/api/users';
 
 export default function endMeeting() {
   const REDIS_CONFIG = Meteor.settings.private.redis;
   const CHANNEL = REDIS_CONFIG.channels.toAkkaApps;
   const EVENT_NAME = 'LogoutAndEndMeetingCmdMsg';
   const { meetingId, requesterUserId } = extractCredentials(this.userId);
 
+  const endedBy = Users.findOne({
+    meetingId,
+    userId: requesterUserId,
+  }, { fields: { name: 1 } });
+
+  if (endedBy) {
+    const selector = {
+      meetingId,
+    };
+
+    const modifier = {
+      $set: {
+        meetingEndedBy: endedBy.name,
+      },
+    };
+
+    Meetings.update(selector, modifier);
```

Comment:
We should update the collections with information coming from BBB core (akka-apps), rather than before informing BBB core about a change. Please obtain the info for who ended the meeting in the modifier or handler, probably `handleMeetingEnd` would be a good spot

Reply:
applied suggested changes in 826c420